### DISCUSSION TITLE
chattheatre.net domain down?

### DISCUSSION THREAD

**Discussion by shentino** [ChatTheatre/Community#12] [General]
Is something screwy going on with the DNS?

The chattheatre.net domain doesn't appear to be resolving.  Is it broken for anyone else or just me?

**Comment by ChristopherA** [comment]
http://chattheatre.com redirects to https://www.chattheatre.com/ working for me.

I think the problem is https://chattheatre.com/ as there is no TLS cert for that.

**Reply by shentino** [reply]
Thanks, I didn't notice the switch from .net to .com, everything's working fine now and I just finished the OS update on gables.
